migrations/20200115142547_pil_fee_waivers.js

```diff
@@ -0,0 +1,26 @@
+
+exports.up = function(knex) {
+  return knex.schema
+    .createTable('fee_waivers', table => {
+      table.uuid('id').primary().defaultTo(knex.raw('uuid_generate_v4()'));
+
+      table.integer('year').notNull();
+      table.index('year');
+
+      table.uuid('pil_id').references('id').inTable('pils').notNull();
+      table.index('pil_id');
+
+      table.integer('establishment_id').references('id').inTable('establishments').notNull();
+      table.index('establishment_id');
+
+      table.text('comment');
+      table.uuid('profile_id').references('id').inTable('profiles').notNull();
+
+      table.timestamps(true, true);
```

Comment:
Does the first param for `.timepstamps` not have any effect on Postgres? Asking cause we seem to use `(false, true)` in all the other migrations which tells it not to use timestamps, but looking at the actual db they're `timestamptz` anyway, so I'm not sure it does.